lms/djangoapps/instructor/views/tools.py

```diff
@@ -60,15 +60,14 @@ def wrapper(request, course_id):
 def bulk_email_is_enabled_for_course(course_id):
     """
     Staff can only send bulk email for a course if all the following conditions are true:
-    1. Bulk email feature flag is on.
+    1. BulkEmailFlag is enabled.
     2. It is a studio course.
     3. Bulk email is enabled for the course.
     """
-
-    bulk_email_enabled_globally = (settings.FEATURES['ENABLE_INSTRUCTOR_EMAIL'] is True)
+    import rpdb; rpdb.set_trace()
```

Comment:
@dianakhuang @cahrens I was able to get a command-line debugging session running for bokchoy servers using this tool, does it seem worth adding to the base edx requirements?
- Terminal 1: `paver test_bokchoy --fasttest --serversonly`
- Terminal 2: `paver test_bokchoy -t lms/test_lms_instructor_dashboard.py:BulkEmailTest --testsonly`
- Terminal 3 (as vagrant base user): `telnet localhost 4444`